CI and packaging is failing due to outdated netbeans link

New builds are failing due to depretaced `netbeans-vm` instance.

```
BUILD FAILED
/build/autopsy/src/autopsy-4.17.0/build.xml:8: The following error occurred while executing this line:
/build/autopsy/src/autopsy-4.17.0/nbproject/build-impl.xml:41: The following error occurred while executing this line:
/build/autopsy/src/autopsy-4.17.0/nbproject/platform.xml:27: java.net.ConnectException: Connection timed out (Connection timed out)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:607)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:284)
	at sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:173)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:463)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:558)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1162)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1056)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:167)
	at org.apache.tools.ant.taskdefs.Get$GetThread.openConnection(Get.java:766)
	at org.apache.tools.ant.taskdefs.Get$GetThread.get(Get.java:676)
	at org.apache.tools.ant.taskdefs.Get$GetThread.run(Get.java:666)

Total time: 2 minutes 9 seconds
```

For more info, [here][1]

[1]: https://issues.apache.org/jira/browse/INFRA-21200

--- a/nbproject/platform.properties
+++ b/nbproject/platform.properties
@@ -7,9 +7,9 @@ suite.dir=${basedir}
 nbplatform.active=download
 nbplatform.active.dir=${suite.dir}/netbeans-plat/${netbeans-plat-version}
 harness.dir=${nbplatform.active.dir}/harness
-bootstrap.url=https://netbeans-vm.apache.org/uc/${netbeans-plat-version}/tasks.jar
+bootstrap.url=https://netbeans-vm1.apache.org/uc/${netbeans-plat-version}/tasks.jar
 # Where we get the platform from. To see what versions are available, open URL in browser up to the .../updates part of the URL
-autoupdate.catalog.url=https://netbeans-vm.apache.org/uc/${netbeans-plat-version}/updates.xml.gz
+autoupdate.catalog.url=https://netbeans-vm1.apache.org/uc/${netbeans-plat-version}/updates.xml.gz
 cluster.path=\
     ${nbplatform.active.dir}/harness:\
     ${nbplatform.active.dir}/java:\

